ui/src/shared/components/Dygraph.js

```diff
@@ -30,9 +31,9 @@ class Dygraph extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      isSynced: false,
       isHidden: true,
       staticLegendHeight: null,
+      isNotHovering: true,
```

Comment:
Generally like to keep boolean values in the 'affirmative'.  If we do ever need to negate `!isNotHoevering` reads a bit funny.  No one likes double negatives.